tests/acceptance/support/sandbox.rs

```diff
@@ -44,19 +44,24 @@ impl CacheBuilder {
 
         // write expiry file
         let one_day = Duration::from_secs(24 * 60 * 60);
-        let expiry_date = HttpDate::from(if self.expired {
+        let expiry_date = Expires::from(if self.expired {
             SystemTime::now() - one_day
         } else {
             SystemTime::now() + one_day
         });
+
+        let mut header_values = Vec::with_capacity(1);
+        expiry_date.encode(&mut header_values);
+        let encoded_expiry_date = header_values.first().unwrap();
```

Comment:
Out of curiosity, since this pattern is… kind of annoying 😂 is there a migration guide or anything so I could get my head around some of the reasoning behind what they chose API-wise? (I poked around but only a tiny bit! Definitely in “lazy maintainer” mode this Saturday morning.)

Reply:
I think the design would try to reduce allocations in the library.